src/Redux/mainPage.js

```diff
@@ -0,0 +1,55 @@
+// import { createSlice } from "@reduxjs/toolkit/dist/createSlice";
+
```

Comment:
- [ ] Kindly remove all unused or unnecessary codes from your file. It helps keep the codebase clean. 👍